exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,31 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
+    return "true";
+  } else {
+    return "false";
+  }
+}
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) {
+    return "true";
+  } else {
+    return "false";
+  }
+}
+
+function isShortName(name) {
+  if (name === "Daniel") {
+    return "true";
+  }
+}
+
+// function startsWithD(name) {
```

Comment:
Very close!
Check your spelling and cAsInG of the method name, and use MDN as a reference when stuck
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#instance_methods)